manifests/get.pp

```diff
@@ -41,7 +41,7 @@
   $exclude       = undef,
   $include       = undef,
   $exclude_first = true,
-  $keyfile       = undef,
+  $keyfile       = "/home/${user}/.ssh/id_rsa",
```

Comment:
You can't use another parameter as part of a parameter default, as it is not yet in-scope. Only `$name`